src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/PrisonService.kt

```diff
@@ -47,12 +44,12 @@ class PrisonService(
   fun getPrisonApiSourceData(prisonerDetails: PrisonerDetails, activeDataOnly: Boolean = true): PrisonApiSourceData {
     val sentenceAndOffences = getSentencesAndOffences(prisonerDetails.bookingId, activeDataOnly)
     val bookingAndSentenceAdjustments = getBookingAndSentenceAdjustments(prisonerDetails.bookingId, activeDataOnly)
-    val bookingHasFixedTermRecall = sentenceAndOffences.any { isSupported(it.sentenceCalculationType) && from(it.sentenceCalculationType).recallType?.isFixedTermRecall == true }
+    val bookingHasFixedTermRecall = sentenceAndOffences.any { from(it.sentenceCalculationType).recallType?.isFixedTermRecall == true }
     val (ftrDetails, returnToCustodyDate) = getFixedTermRecallDetails(prisonerDetails.bookingId, bookingHasFixedTermRecall)
-    val bookingHasAFine = sentenceAndOffences.any { isSupported(it.sentenceCalculationType) && from(it.sentenceCalculationType).sentenceClazz == AFineSentence::class.java }
+    val bookingHasAFine = sentenceAndOffences.any { from(it.sentenceCalculationType).sentenceType == uk.gov.justice.digital.hmpps.calculatereleasedatesapi.enumerations.SentenceType.AFine }
```

Comment:
I think something may have gone funny in intellij here. The enum SentenceType should be imported

Reply:
Agree - resolving in new commit